internal/ingress/controller/store/store.go

```diff
@@ -532,42 +532,47 @@ func New(
 			// updates to configuration configmaps can trigger an update
 			if key == configmap || key == tcp || key == udp {
 				recorder.Eventf(cm, corev1.EventTypeNormal, "CREATE", fmt.Sprintf("ConfigMap %v", key))
+
 				if key == configmap {
 					store.setConfig(cm)
 				}
-				updateCh.In() <- Event{
-					Type: ConfigurationEvent,
-					Obj:  obj,
-				}
+			}
+
+			updateCh.In() <- Event{
+				Type: ConfigurationEvent,
+				Obj:  obj,
```

Comment:
This is a nop change right because `key` is always a configmap anyway?

Reply:
Not sure what do you mean. We need an `<- Event` to trigger a sync.

Reply:
asking in another way: why did you move this code block to here outside of the if statement above?